src/tainting/Dataflow_tainting.ml

```diff
@@ -793,8 +842,7 @@ and check_tainted_expr env exp : Taints.t * Lval_env.t =
         handle_taint_propagators { env with lval_env } (`Exp exp) taints
       in
       let taints = Taints.union taints taints_propagated in
-      let findings = findings_of_tainted_sinks env taints sinks in
-      report_findings env findings;
+      check_orig_if_sink env exp.eorig taints;
```

Comment:
i like this! it confused me before when both relevant lines were relatively distributed